package.json

```diff
@@ -11,9 +11,9 @@
     "clicktest": "mocha --config .mochaclicktest.json",
     "lint": "eslint . bin/*",
     "format": "npm run lint -- --fix",
-    "build": "grunt",
-    "watch": "grunt watch",
-    "package": "grunt package",
+    "build": "node ./scripts/build.js",
+    "watch": "nodemon -C --exec \"npm run build\" -e js,less -w public/source -w source/ -w components/ -w public/less -i \"*.bundle.js\"",
```

Comment:
Why is `source/` being watched?

Reply:
Not quite sure, it was there before. Maybe because of the `source/address-parser.js`?

Reply:
Or because of babel? I will remove it.